config_example.sh

```diff
@@ -622,11 +622,23 @@ set -x
 
 # Set a single config variable AGENT_E2E_TEST_SCENARIO to create a cluster for the different scenarios 
 # i.e. Single Node Openshift(SNO), Highly Available (HA) or Compact cluster.
-# The only supported values for AGENT_E2E_TEST_SCENARIO are COMPACT_IPV4, COMPACT_IPV6, HA_IPV4, HA_IPV6, SNO_IPV4 and SNO_IPV6.
+# The only supported values for AGENT_E2E_TEST_SCENARIO are 
+# - COMPACT_IPV4
+# - COMPACT_IPV6
+# - DHCP_COMPACT_IPV4
+# - DHCP_COMPACT_IPV6
+# - HA_IPV4
+# - HA_IPV6
+# - DHCP_HA_IPV4
+# - DHCP_HA_IPV6 
+# - SNO_IPV4
+# - SNO_IPV6
+# - DHCP_SNO_IPV4
+# - DHCP_SNO_IPV6
 # When set, the code internally sets other low level details such as disk size, memory, number of masters and workers,
 # cpu and ip stack.
 # This config variable is used only by the agent based installer and is required.
-# export AGENT_E2E_TEST_SCENARIO=HA_IPV4
+# export AGENT_E2E_TEST_SCENARIO=DHCP_COMPACT_IPV4
```

Comment:
In general I like the idea of extending the scenario naming convention to include other salient factors, but I think that topology type should come first (ie `COMPACT_IPV4_DHCP` or `SNO_IPV4_DHCP`)

Reply:
If using, `AGENT_MANIFESTS` then we don't need to set like `AGENT_E2E_TEST_SCENARIO=DHCP_COMPACT_IPV4`

Reply:
If not using the `AGENT_MANIFESTS` then I'd keep a consistent naming convention (by adding the `_DHCP` suffix at the end of `AGENT_E2E_TEST_SCENARIO`). This would also make the variable more readable and minimize the impacts when the `AGENT_MANIFESTS` will be introduced